spec/controllers/versions_controller_spec.rb

```diff
@@ -13,6 +13,16 @@
       it "redirects to login" do
         expect(response).to redirect_to new_sessions_path
       end
+
+      context "when an external user" do
+        before(:each, user: :external_user) do
```

Comment:
I don't think this will work in a controller test. You need to create a `mock_external_user` method that sets up the session like https://github.com/ministryofjustice/peoplefinder/blob/main/spec/support/login.rb#L7

Reply:
External user now mocked and context properly indented.